include/object-with-language-interface.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+/**
+ * Interface to use for object types that support at least one language.
+ *
+ * @since 3.3
+ */
+interface PLL_Object_With_Language_Interface {
+
+	/**
+	 * Returns the language taxonomy name.
+	 *
+	 * @since 3.3
+	 *
+	 * @return string
+	 *
+	 * @phpstan-return non-empty-string
+	 */
+	public function get_tax_language();
+
+	/**
+	 * Adds hooks.
+	 *
+	 * @since 3.3
+	 *
+	 * @return void
```

Comment:
Wouldn't it be better if `init()` return `self`?

Reply:
Not useful currently but why not, for consistency. Done in [9600eb0](https://github.com/polylang/polylang/pull/1149/commits/9600eb061ecbd769b1d1e4292408f3b74b747fb9).